### PR TITLE
fixed serializer naming strategy for tests

### PR DESCRIPTION
**Pull request by danrot**
__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none
| Related PRs      | https://github.com/sulu-io/sulu-standard/pull/607
| BC breaks        | none
| Documentation PR | none

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@ CHANGELOG for Sulu
 ==================
 
 * dev-develop
+    * BUGFIX      #1986 [All]             Fixed naming of serializer properties
```
Comment (chirimoya) [inline CHANGELOG.md]: indent

**Comment by wachterjohannes** [comment]
@danrot tests failing code looks good

**Comment by danrot** [comment]
Ok, looks like the `@` for defining services is only allowed in quotes in PHP7. However, there is still this problem with jackrabbit: https://travis-ci.org/sulu-io/sulu/jobs/107959731#L1253

@wachterjohannes Have you already solved that somehow?

**Comment by wachterjohannes** [comment]
@danrot not really ...

**Pull request merged**